package.json

```diff
@@ -26,9 +26,9 @@
     "devDependencies": {
         "@types/debug": "^4.1.7",
         "@types/node": "^12.20.55",
-        "@types/node-fetch": "^2.6.2",
+        "@types/node-fetch": "2.6.2",
```

Comment:
Why is the requirement changed for the types but not for the dependency itself? Shouldn’t be both or neither changed?

Reply:
No. The types are broken so we pin them for us until they're fixed. That doesn't mean we cannot permit library users to use newer versions.